crypto/Cargo.toml

```diff
@@ -4,14 +4,15 @@ version = "0.1.0"
 authors = ["debris <marek.kotewicz@gmail.com>"]
 
 [dependencies]
-blake2-rfc = { git = "https://github.com/gtank/blake2-rfc.git", branch = "persona" }
+bellman = "0.1"
```

Comment:
thought it is better to move all crypto-specific dependencies from `verification` here